### PR TITLE
fix version in tarball

### LINKED ISSUE #337: Release artifact tarballs are missing qcengine/_version.py replacement
**Describe the bug**
Psi4's FindPythonModule cmake module fails to find qcengine because it can't figure out the version:

```
Setting up python3-qcengine (0.21.0-1) ...
[...]
-- Suitable qcengine could not be located, Building qcengine instead.
```

The Cache has:

```
psi4-1.5# grep -i qcengine builddir/CMakeCache.txt 
//Location of Python module qcengine
PY_qcengine:STRING=/usr/lib/python3/dist-packages/qcengine
//Version of Python module qcengine
qcengine_VERSION:STRING=0+unknown
```

Same if I run `qcengine info`:
```
psi4-1.5# qcengine info | grep QCEngine
QCEngine:    0+unknown
```

This seems to be because the tarball if downloaded from Github still has (contrary to the QCElemental tarball, but I couldn't figure out why) the placeholders:

```
$ grep git_ QCEngine-0.21.0/qcengine/_version.py | head -4
    git_refnames = "$Format:%d$"
    git_full = "$Format:%H$"
    git_date = "$Format:%ci$"
    keywords = {"refnames": git_refnames, "full": git_full, "date": git_date}
```

I also checked (randomly) the 0.13.0 tarball, and it's the same there.

QCElemental's tarball e.g. has:

```
$ grep git_ QCElemental-0.24.0/qcelemental/_version.py | head -4
    git_refnames = " (HEAD -> master, tag: v0.24.0)"
    git_full = "cabec4a7d1095b656320f2c842f0e132149e4bd1"
    git_date = "2021-11-18 14:41:00 -0500"
    keywords = {"refnames": git_refnames, "full": git_full, "date": git_date}
```

**To Reproduce**

```
wget -O - https://github.com/MolSSI/QCEngine/archive/refs/tags/v0.21.0.tar.gz 2> /dev/null | zcat | tar xO | grep git_ | head -5
__git_revision__ = get_information("git_revision")
    git_refnames = "$Format:%d$"
    git_full = "$Format:%H$"
    git_date = "$Format:%ci$"
    keywords = {"refnames": git_refnames, "full": git_full, "date": git_date}
```

**Expected behavior**

Same as QCElemental

### PR DESCRIPTION
**Pull request by loriab**
## Description
 fixes #337 confirmed

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -17,6 +17,15 @@ Changelog
 .. +++++++++
 
 
+v0.21.1 / 2022-MM-DD
```
Comment (loriab) [inline docs/source/changelog.rst:20]:
```suggestion
v0.21.1 / 2022-01-06
```

File: setup.cfg [hunk 2]
```diff
@@ -2,9 +2,7 @@
 
 [coverage:run]
 # .coveragerc to control coverage.py and pytest-cov
-# Omit the test directory from test coverage
 omit =
-    #*/tests/*
```
Comment (WardLT) [inline setup.cfg:7]: Why the change here?

Comment (dotsdl) [reply setup.cfg:7]: Looks like just some cleanup tossed in; was already a line with no effect.

**Review by dotsdl — APPROVED** [review]
Looks good @loriab! Merging!
